packages/client/src/exchanges/postExchanges.ts

```diff
@@ -0,0 +1,22 @@
+import { adaptError } from '../helpers/client/formatError.js';
```

Comment:
This file should be called `postExchange` since it will create a single exchange and not exchanges.
Please, review this and change all references to `postExchanges` (including type names) to `postExchange`.

Reply:
The endpoint is able to create multiple exchanges. The data it receives has a property called `ExchangeGroups` which is an array of items to exchange, allowing the user to exchange multiple items at the same time.

Reply:
But then we have an inconsistency here: The type of the return value of this client is `Exchange` not `Exchanges`. I think the question here is what is considered an exchange? If we see all the items that the user want to exchange as a single entity, then it is correct to treat it in the singular and not plural.

Reply:
I think in this case both names would be acceptable, but I understand what you are saying. I've changed all occurrences to be singular and not plural.